docs/loadbalancer-examples.md

```diff
@@ -77,3 +77,44 @@ You can now access your service via the provisioned load balancer
 ```bash
 curl -i http://51.15.224.149
 ```
+
+## Deploy a loadbalancer with specific IP
+
+Instead of creating a loadbalancer service with an ephemeral IP (which change each time your delete/create the service), you can use a reserved loadbalancer address.
```

Comment:
```suggestion
Instead of creating a loadbalancer service with an ephemeral IP (which changes each time your delete/create the service), you can use a reserved loadbalancer address.
```